code/modules/mob/living/simple_animal/hostile/spider_queen.dm

```diff
@@ -0,0 +1,108 @@
+/mob/living/simple_animal/hostile/spider_queen
+	name = "colossal spider"
+	desc = "Furry and brown, it makes you shudder to look at it. This one has deep red eyes."
+	icon = 'icons/mob/npc/spider_queen.dmi'
+	icon_state = "spider_queen"
+	icon_living = "spider_queen"
+	icon_dead = "spider_queen_dead"
+	speak_emote = list("chitters")
+	emote_hear = list("chitters")
+	speak_chance = 5
+	turns_per_move = 5
+	see_in_dark = 10
+	see_invisible = SEE_INVISIBLE_NOLIGHTING
+	meat_type = /obj/item/reagent_containers/food/snacks/xenomeat
+	response_help  = "pets"
+	response_disarm = "gently pushes aside"
+	response_harm   = "pokes"
+	stop_automated_movement_when_pulled = 0
+	maxHealth = 700
+	health = 700
+	melee_damage_lower = 25
+	melee_damage_upper = 30
+	heat_damage_per_tick = 20
+	cold_damage_per_tick = 20
+	faction = "spiders"
+
+	mob_swap_flags = HUMAN|SIMPLE_ANIMAL|SLIME|MONKEY
+	mob_push_flags = ALLMOBS
+
+	attacktext = "bit"
```

Comment:
```suggestion
	attacktext = "bites"
```

Reply:
![image](https://user-images.githubusercontent.com/11579997/72087784-ce685680-32e7-11ea-97d2-304c3f34e927.png)
Bit is correct here.